ogre_mesh_viewer.py

```diff
@@ -313,6 +313,63 @@ def preRenderTargetUpdate(self, evt):
             entity.getSubEntities()[highlight].setMaterial(self.app.highlight_mat)
             self.highlighted = highlight
 
+        if ImGui.CollapsingHeader("Transform"):
```

Comment:
I think this should go at the very end after "Bounds".

Also do not display this when only a mesh is loaded. In this case it is always attached to the root node and the transform is identity. So this tab does nothing.